plugins/dstat_nvidia_gpu.py

```diff
@@ -0,0 +1,41 @@
+### Author: Vasilis Vryniotis <bbriniotis@datumbox.com>
+
+class dstat_plugin(dstat):
+    """
+    Total GPU usage for NVIDIA cards. Requires the nvidia-ml-py package.
+
+    Usage: dstat --nvidia-gpu
+    """
+
+    def __init__(self):
+        self.name = 'gpu nvidia'
+        self.nick = ('gpu',)
+        self.vars = ('gpu',)
+        self.type = 'p'
+        self.width = 5
+        self.scale = 1
+
+    def check(self):
+        import pynvml
```

Comment:
Just noticed that we already catch the import error in Dstat itself:
```
Module dstat_nvidia_gpu failed to load. (No module named pynvml)
```

So if you consider that good enough, I don't think we have to do something special.
(Maybe I should improve that error to state: `The "pynvml library is missing from this system.`)